app/table-config/housing/gross-rent.js

```diff
@@ -105,6 +105,13 @@ export default [
           ],
         },
       },
+      {
+        column: 'cv',
+        aggregator: calculator,
```

Comment:
You'll want to use `formula` as the aggregator. The calculator is a more generic language. 

Also, if you'll notice, `formula` hasn't been imported at the top of the file, so it won't work. You can import like this:
`import formula from '../../utils/formula';`